### PR TITLE
Added TextStyle to CommandCards

### PR DESCRIPTION
**Pull request by mazenhalawi**
I added TestStyle to the CommandCard text to enable the developer to use other background colors that won't interfere with the text. Also, the text is quite small so using TextStyle the developer is free to style the text the way he wants with Size, Weight, Color...

### CODE REVIEW

File: example/ios/Flutter/AppFrameworkInfo.plist [hunk 1]
```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>9.0</string>
```
Comment (radikris) [inline example/ios/Flutter/AppFrameworkInfo.plist:24]: I don't think these iOS config editings are neccessary for the pull request, please remove from the changes

**Review by radikris — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!
If you solve my comment regarding to the iOS config changes, I will approve your pull request, thanks again!

File: example/ios/Flutter/AppFrameworkInfo.plist (hunk 1 above)
Comment (mazenhalawi) [reply example/ios/Flutter/AppFrameworkInfo.plist]: Sorry I didnt realize those files changed. I just reverted the ios folder in the example to the way it was before my changes.

Comment (mazenhalawi) [reply example/ios/Flutter/AppFrameworkInfo.plist]: hey @radikris, did you check the changes I made?

**Comment by mazenhalawi** [comment]
hello @radikris, its been a while, do you think you can push those changes so i can use the features i have implemented please?

**Comment by radikris** [comment]
@mazenhalawi Hi, sorry for my delay, I am back, and alright, it's merged

**Pull request merged**